modules/oe_whitelabel_contact_forms/oe_whitelabel_contact_forms.module

```diff
@@ -74,13 +74,22 @@ function oe_whitelabel_contact_forms_preprocess_status_messages(&$variables) {
       if ($field->isEmpty() || !$field->access()) {
         continue;
       }
-      $value = 'value';
-      if ($field->getFieldDefinition()->getType() == 'skos_concept_entity_reference') {
-        $value = 'target_id';
+
+      $definition = '';
+      if ($field->getFieldDefinition()->getType() === 'skos_concept_entity_reference') {
+        /** @var \Drupal\rdf_skos\Entity\Concept[] $concept_entities */
+        $concept_entities = $field->referencedEntities();
+        if (isset($concept_entities[0])) {
+          $definition = [['label' => $concept_entities[0]->label()]];
+        }
```

Comment:
If `$concept_entities[0]` is not set, we would rather skip (= `continue;`) this field instead of showing empty.
On the other hand, this case cannot really happen because we checked `->isEmpty()` above.

I imagine @brummbar or @drishu would say we don't even need to check `isset()` because we fully rely on the `->isEmpty()` check.

Btw In oe_theme, we call `$field->view(['label' => 'hidden', ...])` instead of what we do here.
The result is a render element.
But if we use this, we would want to get rid of field wrappers, ideally.
But for now let's use `->label()`, we can rethink later.

Reply:
Btw: As long as the value is a simple string, we can skip the `[[..]]` array levels here, and also the 'label'.
This is due to how bcl-item-list works:
- If `$item['definition']` is "iterable" (always true for arrays), then the format has to be `$item['definition'][]['label'] = 'hello'`.
- If `$item['definition']` is a string or non iterable object, then the format can be `$item['definition'] = 'hello'`, so we save multiple array levels. This is what we already do for other values right now. But this only works because these are simple strings instead of render elements.

Reply:
yes, for me the empty check is sufficient

Reply:
Ok, so let's

- Drop the `[[`.
- Drop the `isset` check, and assume that the skos entity exists.